### ISSUE TITLE
Error smtp after fresh install code 111

### ISSUE THREAD

**Issue by valderaplets** [YunoHost-Apps/bitwarden_ynh#73]
hi, i have this after a fresh install : 

- test smtp
- invite email

```
May 27 07:43:05 yunohost bitwarden_rs[1015]: Error: LetreErr.
May 27 07:43:05 yunohost bitwarden_rs[1015]: [CAUSE] Io(
May 27 07:43:05 yunohost bitwarden_rs[1015]:     Os {
May 27 07:43:05 yunohost bitwarden_rs[1015]:         code: 111,
May 27 07:43:05 yunohost bitwarden_rs[1015]:         kind: ConnectionRefused,
May 27 07:43:05 yunohost bitwarden_rs[1015]:         message: "Connection refused",
May 27 07:43:05 yunohost bitwarden_rs[1015]:     },
May 27 07:43:05 yunohost bitwarden_rs[1015]: )
```

The other apps can send email so i don't think it's postfix

**Comment by valderaplets** [comment]
After a new install, and after allow signup without email verification, i have :

```
Jun 02 09:06:32 MYDOMAIN bitwarden_rs[22942]: Error: Error sending welcome email: LetreErr.
Jun 02 09:06:32 MYDOMAIN bitwarden_rs[22942]: [CAUSE] Io(
Jun 02 09:06:32 MYDOMAIN bitwarden_rs[22942]:     Os {
Jun 02 09:06:32 MYDOMAIN bitwarden_rs[22942]:         code: 111,
Jun 02 09:06:32 MYDOMAIN bitwarden_rs[22942]:         kind: ConnectionRefused,
Jun 02 09:06:32 MYDOMAIN bitwarden_rs[22942]:         message: "Connection refused",
Jun 02 09:06:32 MYDOMAIN bitwarden_rs[22942]:     },
Jun 02 09:06:32 MYDOMAIN bitwarden_rs[22942]: )
Jun 02 09:06:47 MYDOMAIN bitwarden_rs[22942]: Error: Error sending new device email: LetreErr.
Jun 02 09:06:47 MYDOMAIN bitwarden_rs[22942]: [CAUSE] Io(
Jun 02 09:06:47 MYDOMAIN bitwarden_rs[22942]:     Os {
Jun 02 09:06:47 MYDOMAIN bitwarden_rs[22942]:         code: 111,
Jun 02 09:06:47 MYDOMAIN bitwarden_rs[22942]:         kind: ConnectionRefused,
Jun 02 09:06:47 MYDOMAIN bitwarden_rs[22942]:         message: "Connection refused",
Jun 02 09:06:47 MYDOMAIN bitwarden_rs[22942]:     },
Jun 02 09:06:47 MYDOMAIN bitwarden_rs[22942]: )
Jun 02 09:06:47 MYDOMAIN bitwarden_rs[22942]: Error: ###########################################################
Jun 02 09:06:47 MYDOMAIN bitwarden_rs[22942]:     '/notifications/hub' should be proxied to the websocket server or notifications won't work.
Jun 02 09:06:47 MYDOMAIN bitwarden_rs[22942]:     Go to the Wiki for more info, or disable WebSockets setting WEBSOCKET_ENABLED=false.
Jun 02 09:06:47 MYDOMAIN bitwarden_rs[22942]:     ###########################################################################################
```

and after click on "Check your account email address to unlock all features" i have

```
Jun 02 10:23:11  bitwarden_rs[22942]: Error: Error sending delete account email: LetreErr.
Jun 02 10:23:11  bitwarden_rs[22942]: [CAUSE] Io(
Jun 02 10:23:11  bitwarden_rs[22942]:     Os {
Jun 02 10:23:11  bitwarden_rs[22942]:         code: 111,
Jun 02 10:23:11  bitwarden_rs[22942]:         kind: ConnectionRefused,
Jun 02 10:23:11  bitwarden_rs[22942]:         message: "Connection refused",
Jun 02 10:23:11  bitwarden_rs[22942]:     },
Jun 02 10:23:11  bitwarden_rs[22942]: )
```

**Comment by valderaplets** [comment]
Hello, I solved the problem by putting 127.0.0.1 instead of localhost, although my other applications didn't encounter this problem.
